example_scenes/manim.cfg

```diff
@@ -1,4 +1,5 @@
 [log.color]
+#It is parsed by Rich Colour Parse `rich.color`
```

Comment:
```suggestion
# The colors here are parsed by rich.color , and are not related
# to the colours defined in constants.py's COLOR_MAP
```
I think this would be a bit clearer. The user may think that they will be able to input colours defined in Manim's own colour palette.